src/main/java/use_cases/monthly_menu/MonthMenuOD.java

```diff
@@ -0,0 +1,85 @@
+package use_cases.monthly_menu;
+
+import entities.Category;
```

Comment:
Is the OutputData allowed to import the entity packages? Generally we should be using general Object types in the OutputData. I believe the SessionStorage in the InputData is such an exception since we have no other way of accessing the rest of the stuff without it, but these could be replaced (and probably should).

Reply:
Same as mentioned in MonthMenuV review.